### ISSUE TITLE
Florisboard: Gestures not working properly

### ISSUE THREAD

**Issue by ektosgit** [florisboard/florisboard#2634]
### Short description

Hello Florisboard Team, 

i have an issue after updating to android 14 vanilla (nokia x30). App gestures do not work properly anymore. 
i use Swipe up on the keyboard to shift, swipe down to close the keyboard and swipe left and right to change languages. This is a hustle after updating to android 14. No matter what sensitivity settings i use (swipe velocity and swipe distance) it rarerly works and only when i manage to touch the screen with the very tip of my finger and swipe very short distances and very fast. Please fix this. This are the most functions i use on a keyboard.

### Steps to reproduce

swiping on the keyboard with active gestures. On vanila android devices
.
### FlorisBoard Version

0.4.1

### Install Source

GitHub

### Device

nokia x30

### Android

14 stock (vanilla)

**Comment by Keyaku** [comment]
Since I've never tested this keyboard before Android 14, I was wondering if it was implemented/working at all; apparently it's just a bug.
I want to get rid of the Globe icon on the keyboard because, in my attempts to avoid it, I keep missing the spacebar. It's smaller than I want and off-centered thanks to the extra keys down there.

I wonder if this is an easy fix.

**Comment by itisafrickinhighlander** [comment]
I have recently changed my Samsung phone with a Pixel phone and I have just spent a while trying to wrap my mind around this exact issue. Just like it was described, it is as if it requires a very specific small surface (a tip of your finger) fast flick (e.g. it almost never registers when I use my thumb naturally when holding the phone normally). I actually didn't have this issue with that Samsung phone (also S21 like @Keyaku , but the FE model) and only experience it now on Pixel 8A.

**Comment by ektosgit** [comment]
@itisafrickinhighlander, yeah sadly the issue persists on android 15 and florisboard version 0.4.6, i have a fraling that sadly nobody cares🥲.
The issue i presume happens mostly on vanilla androids, like hmd devices, google devices, nothing devices and fairphones.

**Comment by tsiflimagas** [comment]
Thanks for the report! This issue has been confirmed. I understand that it's been around for long, but the backlog is quite long. In the meantime, let's keep discussion at #2515 (#2429 was the first report).
